synapse/api/auth.py

```diff
@@ -260,6 +266,11 @@ def get_user_by_req(
 
             request.authenticated_entity = user.to_string()
 
+            opentracing.set_tag("authenticated_entity", user.to_string())
```

Comment:
Unfortunately this wouldn't work because there are places we set authenticated entity before we have an active span available. For example we only want to start this span https://github.com/matrix-org/synapse/blob/46e41ee08d40750f6992b9479be18006162bee32/synapse/federation/transport/server.py#L290 after we've authenticated the request. Which means
https://github.com/matrix-org/synapse/blob/46e41ee08d40750f6992b9479be18006162bee32/synapse/federation/transport/server.py#L147 is done prior to an active span.

Reply:
part of me wonders if that means we should start the span sooner, but fair enough.